lib/ruby-debug-ide/xml_printer.rb

```diff
@@ -205,7 +207,7 @@ def exec_with_allocation_control(value, memory_limit, time_limit, exec_method, o
       inspect_thread.kill
       return result
     rescue MemoryLimitError, TimeLimitError => e
-      print_debug(e.message + "\n" + e.backtrace)
+      print_debug(e.message + "\n" + e.backtrace.join("\n"))
```

Comment:
Is `caller.map {|l| "\t#{l}"}.join("\n")}` the same as `caller.to_a.join("\n")`?